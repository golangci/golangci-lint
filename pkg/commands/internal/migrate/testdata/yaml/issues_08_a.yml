issues:
  # Only to not generate unrelated elements inside golden.
  exclude-use-default: false
  # Only to not generate unrelated elements inside golden.
  exclude-generated: strict
  # Only to not generate unrelated elements inside golden.
  exclude-dirs-use-default: false

  max-issues-per-linter: 0
  max-same-issues: 0
  uniq-by-line: true
