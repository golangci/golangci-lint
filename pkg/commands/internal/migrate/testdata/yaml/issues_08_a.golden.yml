version: "2"
issues:
  max-issues-per-linter: 0
  max-same-issues: 0
  uniq-by-line: true
